apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -1,13 +1,39 @@
-import { Keys } from './consts';
+import { Keys, ROOT_VIEW } from './consts';
 import { TESTPAGE_BUTTONS_SCROLLVIEWER } from '../../TestComponents/Common/consts';
 import { Attribute } from './consts';
 
 const DUMMY_CHAR = '';
+// The E2ETEST_PLATFORM environment variable should be set in the beforeSession hook in the wdio.conf file for the respective platform
+const PLATFORM = process.env['E2ETEST_PLATFORM'];
+if (PLATFORM) {
```

Comment:
Is this still needed? Leftover from testing? When running a test, you need to specify a platform-specific wdio file, so I I don't think this adds value.

Reply:
This is for getting the current running platform within code and the page objects. Is there an existing way of doing this with wdio?

Reply:
Sorry, I meant the if statement + console.log on line 8-10.

Reply:
Friendly bump on this

Reply:
Fixed :)